tests/lib/CookieJar.spec.js

```diff
@@ -13,14 +13,14 @@ describe('CookieJar', () => {
 
     it('can be filled with key info', () => {
         spyOnProperty(document, 'cookie', 'set').and.callFake((/** @type {string} */ cookie) => {
-            expect(cookie.includes(`k=${Dummy.keyInfoCookieEncoded}`)).toBe(true);
+            expect(cookie.valueOf().substr(0, Dummy.keyInfoCookieEncoded.length + 2)).toEqual(`k=${Dummy.keyInfoCookieEncoded}`);
```

Comment:
Häää?

Reply:
Well, let's have an example. Assume `keyInfoCookieEncoded === '101;102` and the cookie starts with `k=101;102;203`. I wanted to avoid that this passes, but turns out I didn't... Damn.